apps/wrf/install_wps_openmpi.sh

```diff
@@ -23,7 +26,7 @@ module load gcc-9.2.0
 
 export HDF5=$(spack location -i hdf5^openmpi)
 export NETCDF=$(spack location -i netcdf-fortran^openmpi)
-export WRF_DIR=${SHARED_APP}/WRF-${WRF_VERSION}
+export WRF_DIR=${SHARED_APP}/${SKU_TYPE}/wrf-openmp/WRF-${WRF_VERSION}
```

Comment:
is it wrf_openmpi or wrf-openmpi ?
Same feedbacks as for mvapich2 regarding reusing APP_DIR

Reply:
wrf-openmpi is correct (not wrf_openmpi).